scripts/enable-resource-locking.sh

```diff
@@ -32,9 +32,12 @@ exclusions=(
   ss-prod-network-rg
 )
 
+RG_LIST_EXEMPT=$(az group list --query "[?tags.exemptFromAutoLock=='true'].name" -o tsv | sort -u )
```

Comment:
why do you need another query for this?

Reply:
the first query only querying the resource list and not group, so if any of the resource from the resource group hasn't got the tag, it will lock the resource group with all the resources which has got the tag.  this is the reason it did not work las time

Reply:
ah why don't we just tag the resources?

Reply:
we can, although I thought if we have resource which couldn't be tagged and if the lock is applied to resource group only, its best to have exemption on the resource group wise.  I can put condition back for the resources as well as for resource group?

Reply:
yeah both is probably better

Reply:
done